tools/travis/.travis.yml

```diff
@@ -0,0 +1,52 @@
+language: php
+
+php:
+  - 5.3
+  - 5.4
+  - 5.5
+  - 5.6
+  - 7.0
+  - nightly
+  - hhvm
+
+env:
+  global:
+    - PATH="$HOME/.composer/vendor/bin:$PATH"
+
+matrix:
+  fast_finish: true
+  include:
+    - php: 5.6
+      env: COMPOSER_FLAGS="--prefer-lowest"
+    - php: 5.6
+      env: SYMFONY_VERSION=2.7.*
+    - php: 5.6
+      env: SYMFONY_VERSION=2.8.*
+    - php: 5.6
+      env: SYMFONY_VERSION=3.0.*
+    - php: 5.6
+      env: SYMFONY_VERSION=3.1.*@dev
+  allow_failures:
+    - php: nightly
+    - env: SYMFONY_VERSION=3.1.*@dev
+
+sudo: false
+
+cache:
+  directories:
+    - $HOME/.composer/cache/files
+
+before_install:
+  - if [ "$TRAVIS_PHP_VERSION" != "hhvm" ]; then phpenv config-rm xdebug.ini; fi;
+
+install:
+  - composer config -g github-oauth.github.com $GITHUB_OAUTH_TOKEN
+  - composer global require phpunit/phpunit:@stable --no-update
+  - composer global update --prefer-dist --no-interaction
+  - if [ "$SYMFONY_VERSION" != "" ]; then composer require "symfony/symfony:${SYMFONY_VERSION}" --no-update; PHPUNIT_OPTION="--testsuite symfony"; fi;
+  - composer update --prefer-dist --no-interaction $COMPOSER_FLAGS
+
+before_script:
+  - if [ "$TRAVIS_PHP_VERSION" != "hhvm" ]; then echo "zend_extension=xdebug.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini; fi;
```

Comment:
Yeah seriously : 

```
- | if [ "$TRAVIS_PHP_VERSION" != "hhvm" ]; 
    then echo "zend_extension=xdebug.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini; 
    fi;
```
is an example of how beautiful it could be come.

And I would argue that this is not just a command line because you make it a line. You're not at home in your terminal hacking something, you're scripting something. So use several lines, long options, and if it grows too much, move all that to a separate shell script that will have syntactic coloration and proper indention.